README-el.md

```diff
@@ -340,9 +349,9 @@ mkdir empty && rsync -r --delete empty/ some-dir && rmdir some-dir
 
 - Είναι εξαιρετικά βοηθητικό ορισμένες φορές το γεγονός ότι μπορείτε να εφαρμόσετε τις πράξεις της τομής, της ένωσης και της διαφοράς σε αρχεία κειμένου με τις εντολές `sort`/`uniq`. Ας πούμε ότι έχετε `a` και `b` αρχεία κείμενου που καμία γραμμή τους δεν επαναλαμβάνεται. Αυτό είναι γρήγορο και δουλεύει σε αρχεία οποιουδήποτε μεγέθους έως πολλά gigabytes. (H `sort` δεν περιορίζεται από τη μνήμη, αν και ίσως χρειαστεί να χρησιμοποιήσετε την επιλογή `-T`, αν ο `/tmp` βρίσκεται σε μικρή κατάτμηση (partition). Επίσης, δείτε την παρατήρηση παραπάνω για τη μεταβλητή `LC_ALL` και την επιλογή `-u` της εντολής `sort` (παραλείπεται παρακάτω ώστε να είναι πιο ξεκάθαρο το παράδειγμα).
 ```sh
-      cat a b | sort | uniq > c   # το c είναι a ένωση b
-      cat a b | sort | uniq -d > c   # c είναι a τομή b
-      cat a b b | sort | uniq -u > c   # το c είναι a διαφορά b
+      sort a b | uniq > c   # το c είναι a ένωση b
+      sort a b | uniq -d > c   # c είναι a τομή b
+      sort a b b | uniq -u > c  # το c είναι η διαφορά του a από το b
```

Comment:
Consider adopting the text of https://github.com/jlevy/the-art-of-command-line/pull/553/commits/1843c1eddc431b570e810fbc59a9c085bdb9176e